LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceCache.cs

```diff
@@ -347,13 +347,13 @@ public record LoRaDeviceCacheOptions
         /// as LoRa sensors are optimized for saving power and
         /// might only send a message, once a week for example.
         /// </summary>
-        public TimeSpan MaxUnobservedLifetime { get; set; }
```

Comment:
This was on purpose not immutable. The goal is the same as with the logger to be able to drive these values through twins and have them updated while running. To do this, they need to be mutable.

Reply:
Fair enough but then what was the thinking behind making the options a record?